react/src/components/first-sample-app/Application.jsx

```diff
@@ -15,6 +17,7 @@ class Application extends React.Component {
         First Sample Application
         <button onClick={() => this.setState({ toggle: !toggle })}> { toggle ? 'Click me again!' : 'Click meeeeee!' } </button>
         <CommonComponent />
+        <SearchBarComponent />
```

Comment:
I suggest that we rename that component (first-sample-app) to `main-component` for example, if we want to use it for the index page.